en/config_vtol/vtol_duo_tailsitter_config.md

```diff
@@ -0,0 +1,94 @@
+# Duo-Tailsitter VTOL Configuration
+
+This is the configuration tuning documentation for a "Duo-Tailsitter VTOL".
+
+A tailsitter is essentially a fixed wing vehicle that takes off in a pitched up configuration (essentially sitting on its tail, hence the name) and then transitions to fixed wing flight by pitch down by 90 degrees to achieve the fixed wing configuration.
+The Duo-Tailsitter variant is so-named because it has two motors (a "quad" tailsitter would have four).
+
+Like other VTOLs, the Duo-Tailsitter has two modes of operation: multicopter (MC) mode and fixed wing (FW) mode.
+In MC mode, lift and roll are dependent on the motors, while the pitch and yaw depend on the ailevons.
```

Comment:
What's an ailevon?

Reply:
Ahaha I meant elevon. Sorry Sorry. Will change that.

Reply:
```suggestion
In MC mode, lift and roll are dependent on the motors, while the pitch and yaw depend on the elevons.
```

Reply:
Might as well just fix it now.